validator_client/src/graffiti_file.rs

```diff
@@ -0,0 +1,165 @@
+use serde_derive::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::fs::File;
+use std::io::{prelude::*, BufReader};
+use std::path::PathBuf;
+use std::str::FromStr;
+
+use bls::blst_implementations::PublicKey;
+use types::{graffiti::GraffitiString, Graffiti};
+
+#[derive(Debug)]
+pub enum Error {
+    InvalidFile(std::io::Error),
+    InvalidLine,
+    InvalidPublicKey,
+    NoDefaultField,
+    InvalidGraffiti(String),
+}
+
+/// Struct to load validator graffitis from file.
+/// The graffiti file is expected to have the following structure
+///
+/// default: Lighthouse
+/// public_key1: graffiti1
+/// public_key2: graffiti2
+/// ...
```

Comment:
would you mind adding docs about this and what the graffiti order of precedence is to the lighthouse book?

Reply:
Good point. Will add docs